### ISSUE TITLE
PlayCanvas Online glTF Viewer does not support Binary glTF format

### ISSUE THREAD

**Issue by cx20** [playcanvas/playcanvas-gltf#25]
I tried displaying two kinds of glTF formats using [PlayCanvas Online glTF Viewer](https://playcanvas.github.io/playcanvas-gltf/viewer/index.html).

Regular glTF file is displayed correctly.
https://playcanvas.github.io/playcanvas-gltf/viewer/index.html?assetUrl=https://cdn.rawgit.com/KhronosGroup/glTF-Sample-Models/master/2.0/Duck/glTF/Duck.gltf

However, Binary glTF file is an error and will not be displayed.
https://playcanvas.github.io/playcanvas-gltf/viewer/index.html?assetUrl=https://cdn.rawgit.com/KhronosGroup/glTF-Sample-Models/master/2.0/Duck/glTF-Binary/Duck.glb

```
Uncaught (in promise) SyntaxError: Unexpected token g in JSON at position 0
Promise.then (async)
(anonymous) @ viewer.js:218
Promise.then (async)
main @ viewer.js:217
onload @ index.html?assetUrl=https://cdn.rawgit.com/KhronosGroup/glTF-Sample-Models/master/2.0/Duck/glTF-Binary/Duck.glb:15
```

I confirmed that I can display correctly when I drag & drop the glb file into Viewer. Therefore, I think that it should be displayed also when using `assetUrl`.

**Comment by willeastcott** [comment]
Fixed in 41a60bae1097bd1d8ae41b5aa3e5ffd8d19d08ed.

**Comment by cx20** [comment]
I confirmed that `localhost` problem was solved in the latest version.
